content/id/docs/contribute/suggesting-improvements.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Menyarankan peningkatan konten
```

Comment:
the sentence seems off in literal Indonesian, how about adding helper "kualitas"?
```suggestion
title: Menyarankan peningkatan kualitas konten
```